wallet/wallet.go

```diff
@@ -16,12 +15,17 @@ import (
 	"go.sia.tech/renterd/api"
 	"go.sia.tech/siad/modules"
 	"go.uber.org/zap"
-	"lukechampine.com/frand"
 )
 
-// BytesPerInput is the encoded size of a SiacoinInput and corresponding
-// TransactionSignature, assuming standard UnlockConditions.
-const BytesPerInput = 241
+const (
+	// BytesPerInput is the encoded size of a SiacoinInput and corresponding
+	// TransactionSignature, assuming standard UnlockConditions.
+	BytesPerInput = 241
+
+	// redistributeBatchSize is the number of outputs to redistribute per txn to
+	// avoid creating a txn that is too large.
+	redistributeBatchSize = 10
```

Comment:
seems on the low side but on the other hand that's not a big issue, it's probably a 5x off of what it could be though?

Reply:
It is quite conservative but I really want to be on the safe side here. For a large renter that wants to create 100 outputs with a total amount of 100k this would still be 10KS per output which might become a problem depending on the state of wallet fragmentation.